api-tests/ff/ipc/test_i002/test_i002.c

```diff
@@ -184,7 +184,7 @@ int32_t client_test_psa_call_with_allowed_type_values(caller_security_t caller _
 {
    int32_t            status = VAL_STATUS_SUCCESS;
    psa_handle_t       handle = 0;
-   int32_t            type[] = {PSA_IPC_CALL, 1, 2, INT32_MAX};
+   int32_t            type[] = {PSA_IPC_CALL, 1, 2, INT16_MAX};
```

Comment:
An `int16_t` array is probably enough. Same for the other file.

Reply:
@LDong-Arm, do you want to submit the path again with the uint16 array?

Reply:
I'm just a reviewer, will let the author do it when he's available

Reply:
> An `int16_t` array is probably enough. Same for the other file.

Yeah, good catch, I will update them.